semgrep/semgrep/semgrep_types.py

```diff
@@ -26,6 +27,74 @@
 YAML_TAINT_MUST_HAVE_KEYS = {"pattern-sinks", "pattern-sources"}
 
 
+class Language(Enum):
+    PYTHON: str = "python"
+    PYTHON2: str = "python2"
+    PYTHON3: str = "python3"
+    JAVASCRIPT: str = "javascript"
+    TYPESCRIPT: str = "typescript"
+    JAVA: str = "java"
+    C: str = "c"
+    GO: str = "go"
+    RUBY: str = "ruby"
+    PHP: str = "php"
+    LUA: str = "lua"
+    CSHARP: str = "csharp"
+    RUST: str = "rust"
+    KOTLIN: str = "kt"
+    YAML: str = "yaml"
+    ML: str = "ml"
+    JSON: str = "json"
+    REGEX: str = "regex"
+    GENERIC: str = "generic"
+
+
+class Language_util:
+    language_to_strs: Dict[Language, List[str]] = {
+        Language.PYTHON: [Language.PYTHON.value, "py"],
+        Language.PYTHON2: [Language.PYTHON2.value],
+        Language.PYTHON3: [Language.PYTHON3.value],
+        Language.JAVASCRIPT: [Language.JAVASCRIPT.value, "js"],
+        Language.TYPESCRIPT: [Language.TYPESCRIPT.value, "ts"],
+        Language.JAVA: [Language.JAVA.value],
+        Language.C: [Language.C.value],
+        Language.GO: [Language.GO.value, "golang"],
+        Language.RUBY: [Language.RUBY.value, "rb"],
+        Language.PHP: [Language.PHP.value],
+        Language.LUA: [Language.LUA.value],
+        Language.CSHARP: [Language.CSHARP.value, "cs", "C#"],
+        Language.RUST: [Language.RUST.value, "Rust", "rs"],
+        Language.KOTLIN: [Language.KOTLIN.value, "Kotlin", "kotlin"],
```

Comment:
would it require significant changes to standardize on lower case and always have methods .lower() incoming language strings?